contracts/contracts/mocks/MockOGN.sol

```diff
@@ -1,9 +1,107 @@
 pragma solidity 0.5.11;
 
-import "./MintableERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20Burnable.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
 
-contract MockOGN is MintableERC20 {
-    uint256 public constant decimals = 18;
-    string public constant symbol = "OGN";
-    string public constant name = "OriginToken";
-}
+import "./WhitelistedPausableToken.sol";
```

Comment:
It does not hurt but I don't think this is necessary. This is used to pause the token. Which shouldn't be related to the spender whitelist.

Reply:
I see the merit here to use the actual OGN methods. Ensures our contract's behavior is correct. The one thing I might do (open for discussion) is add back the `mint(amount)` function that all our mocks have. This way we don't have to change our tests, and the OGNMock has the same minimal subset of methods as our other token mocks.

Reply:
Yeah I wanted to keep the MockOGN contract as close as possible to the live version.  You make a valid point, I don't feel strongly either way. Maybe @DanielVF, @crazybuster or @tomlinton can wight in?